README.md

```diff
@@ -1,2 +1,41 @@
-# controller-sdk-go
-The Go SDK for the Deis Controller API
+# Controller Go SDK
+[![Build Status](https://travis-ci.org/deis/controller-sdk-go.svg?branch=master)](https://travis-ci.org/deis/controller-sdk-go)
+[![Go Report Card](https://goreportcard.com/badge/github.com/deis/controller-sdk-go)](https://goreportcard.com/report/github.com/deis/controller-sdk-go)
+[![codebeat badge](https://codebeat.co/badges/2fdee091-714d-4860-ab19-dba7587a3158)](https://codebeat.co/projects/github-com-deis-controller-sdk-go)
+[![GoDoc](https://godoc.org/github.com/deis/controller-sdk-go?status.svg)](https://godoc.org/github.com/deis/controller-sdk-go)
+
+This is the go sdk for interacting with the [Deis Controller](https://github.com/deis/controller)
+
+### Usage
+
+```go
+import deis "github.com/deis/controller-sdk-go"
+import "github.com/deis/controller-sdk-go/apps"
+```
+
+Construct a deis client to interact with the controller API. Then, get the first 100 apps the user has access to.
+
+```go
+//                    Verify SSL, Controller URL, API Token
+client, _ := deis.New(true, "deis.test.io", "abc123")
```

Comment:
What should the error handling be? I wanted to keep the examples really short (assuming, probably a mistake on my part 😄, that people would look at the godoc to figure out exactly what to do).

Since the code isn't within a method, I don't really know what to do for error handling. I could print the error, but in order to return and prevent further execution I would have to assume return types.